addons/mail/static/src/js/form_renderer.js

```diff
@@ -1,77 +1,151 @@
         }
-        return this._super.apply(this, arguments);
     },
-
     //--------------------------------------------------------------------------
     // Private
     //--------------------------------------------------------------------------
-
     /**
-     * Overrides the function that renders the nodes to return the chatter's $el
-     * for the 'oe_chatter' div node.
+     * @private
+     */
+    async _createChatter() {
+        const chatterLocalId = await this.env.store.dispatch('createChatter', {
+            initialThreadId: this.state.res_id,
+            initialThreadModel: this.state.model,
+        });
+        this._chatterLocalId = chatterLocalId;
+        this._chatterComponent = new Chatter(null, { chatterLocalId });
+        if (this._chatterEl) {
+            /* FIXME {xdu}
+               could be better to mount in "replace" mode but the mount is failing with that mode
+               await this._chatterComponent.mount(this._chatterEl, { position: 'self' });
+            */
+            await this._chatterComponent.mount(this._chatterEl);
+        }
+    },
+    /**
+     * Delete the chatter component
      *
+     * @private
+     */
+    _deleteChatter() {
+        if (this._chatterComponent) {
+            this._chatterComponent.destroy();
+            this._chatterComponent = undefined;
+        }
+        this.env.store.dispatch('deleteChatter', this._chatterLocalId);
+    },
+    /**
+     * Force mount the chatter
+     *
+     * FIXME {XDU} It's a hack : chatterComponent is in a fragment :
+     * - no more in the DOM
+     * - no more correctly connected to the store
+     * We need to retarget it correctly in the DOM to fix that issue
+     * The issue is that the component considers itself as mounted
+     * but is no more added to the view as the super call to render view
+     * re-rendered the view entirely and without this component.
+     * @private
+     */
+    async _forceMountChatterComponent() {
+        this._chatterComponent.__owl__.isMounted = false;
+        /*  FIXME {xdu}
+            could be better to mount in "replace" mode but the mount is failing with that mode
+            await this._chatterComponent.mount(this._chatterEl, { position: 'self' });
+         */
+        if (this._chatterEl) {
+            await this._chatterComponent.mount(this._chatterEl);
+        }
+    },
+   /**
      * @override
      * @private
      */
-    _renderNode: function (node) {
-        var self = this;
-        if (node.tag === 'div' && node.attrs.class === 'oe_chatter') {
-            if (!this.chatter) {
-                this.chatter = new Chatter(this, this.state, this.mailFields, {
-                    isEditable: this.activeActions.edit,
-                    viewType: 'form',
-                });
+   _renderNode(node) {
+       if (node.tag === 'div' && node.attrs.class === 'oe_chatter') {
+           this._hasChatter = true;
+           const $el = $('<div class="o_Chatter_container"/>');
```

Comment:
This classname does not follow our classname convention: this is not the container part of the chatter component.
I suggest something like `o_FormRenderer_chatterContainer`.

Reply:
I agree but I need to put some style on it and for now this style is in `chatter.scss`, I don't know if `o_FormRenderer_chatterContainer` would make me think to search the style in this file. As form renderer is not yet a component I don't think `form_renderer.scss` is good too ... What do you think ?

Reply:
I'm good with having style in `chatter.scss`. Agree that `o_FormRenderer_chatterContainer` suggests that `FormRenderer` is a component, which is misleading, but this is the lesser problem I have thought of.

Reply:
Ok agree :)